### ISSUE TITLE
new feature: post to a list of recipients

### ISSUE THREAD

**Issue by bastlwasti** [andstatus/andstatus#355]
Hello there,

I am looking for some suitable android pump.io client. I like andstatus because it allows me to write offline and then post it later when I got Wifi connection. This is quite important for me, because I don't want to be connected all the time. 

Unfortunately, I miss once important feature in andstatus: I would like to post certain messages, or photos just to a list of recipients, e.g. the list "Family". Or just to myself. I tried all out, but could't figure out how this can be done with andstatus. Seems it is not supported yet, and I am buffled, that no one misses this :-)

Maybe you can add this feature if it really is not supported if you got some time for it.
thank you anyway,

bastlwasti

**Comment by yvolk** [comment]
Hello @bastlwasti. We already have a request to support "Groups" ("Lists" in Twitter). Both groups in GNU Social and Lists in Twitter are read-only feeds, you cannot post to them.

I don't have any concrete info on what "Lists" in Pump.io mean. Could you help me to find this out and post here?

Anyway, will it be enough, for beginning, to allow filling "To" field (recipient id) of a post explicitly? Actually currently only "Direct" message have a recipient.

**Comment by bastlwasti** [comment]
Yes, I think for the beginning, it would be great to have a "To:" field ;-)

"Lists" in pumpio are just an abbreviation for user addresses. When you create a pump.io account some lists e.g. "Family" are created. And you can add some followers to them appropriately. 

I would like to use pump.io or gnusocial ( I didn't finish my decision yet) to post to only myself, or to my family, or to my good friends, or to all I know (="Followers") some notes or a photo. Unfortunately, gnusocial seems to post everything to all, and pumpio seems to me less "twitter"-like, but only in "alpha" stage.

As far as I can see, Andstatus seems to be the best client. The offline mode is great, I can then use it everywhere, type in my thoughts and make a photo, and send it later when I got a wifi. The only missing thing is, that I don't want to post it to all, and maybe adding the location would be great too... hahaha :)

best wishes :)

**Comment by jankusanagi** [comment]
Those lists are just like the "Followers" list.

If AS was able to post to "Followers", posting to those other lists would be exactly the same. It really needs specific To/Cc fields.

**Comment by yvolk** [comment]
@jankusanagi How do I know/lookup IDs of some lists I see some "lists" element here, for example: 

https://github.com/andstatus/andstatus/blob/master/app/src/androidTest/res/raw/pumpio_activity_with_image.json

but its "url":"https://io.jpope.org/api/user/jpope/lists/person" returns "Cannot GET /api/user/jpope/lists/person" only. Maybe because I'm not authenticated?

**Comment by jankusanagi** [comment]
@yvolk your app should only get **your** own lists, from your own server, to which you're authenticated, of course.

BTW, you only need to get the ID's of the lists to give the user the chance to post to those lists, you don't need to fetch the actual lists (the array of person objects itself).
